src/ip/Address.cc

```diff
@@ -352,7 +352,7 @@ Ip::Address::getReverseString(char buf[MAX_IPSTRLEN], int show_type) const
 Ip::Address&
 Ip::Address::operator =(const Ip::Address &s)
 {
-    memcpy(this, &s, sizeof(Ip::Address));
+    memmove(&mSocketAddr_, &s.mSocketAddr_, sizeof(struct sockaddr_in6));
```

Comment:
If the default generated assignment operator works, please use that instead (and check whether we can remove an explicit copy constructor as well).